website/src/server.mjs

```diff
@@ -50,6 +51,16 @@ export function makeServer({
             path.join(router.wwwRootPath, classifiedDirectory.path),
             { currentURI: request.path }
           );
+          const $ = cheerio.load(out);
+          $("script")
+            .filter(function () {
+              return (
+                this.children[0].data === "\n  //\n" ||
```

Comment:
Must fix: This code crashes when visiting the benchmarks page (http://127.0.0.1:9001/benchmarks/):
```
TypeError: Cannot read property 'data' of undefined
    at Node.<anonymous> (file:///Users/strager/Projects/quick-lint-js/website/src/server.mjs:58:34)
    at /Users/strager/Projects/quick-lint-js/website/node_modules/cheerio/lib/api/traversing.js:531:48
    at Array.filter (<anonymous>)
    at filterArray (/Users/strager/Projects/quick-lint-js/website/node_modules/cheerio/lib/api/traversing.js:548:17)
    at LoadedCheerio.filter (/Users/strager/Projects/quick-lint-js/website/node_modules/cheerio/lib/api/traversing.js:542:23)
    at serveDirectoryAsync (file:///Users/strager/Projects/quick-lint-js/website/src/server.mjs:56:14)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```